code-scanning/properties/checkmarx-one.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Checkmarx",
+    "creator": "Checkmarx",
+    "description": "Beat vulnerabilities with more secure code.Scan your code with Checkmarx One and see results in the GitHub code scanning.",
+    "iconName": "checkmarx",
+    "categories": ["Code Scanning","code-quality", "javascript", "python", "java", "php", "c#", "c", "c++", "ruby", "swift", "go", "json", "kotlin", "apex", "scala", "perl"]
```

Comment:
What do mean by code-quality?

Reply:
These categories are used to suggest templates in a better way to users. Currently we don't support `code-quality`. So adding it wouldn't help recommendation engine. You can remove it